ui/qubemanager.ui

```diff
@@ -317,6 +317,19 @@ template</string>
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="show_user">
+        <property name="text">
+         <string>User qubes</string>
+        </property>
+        <property name="checked">
+         <bool>true</bool>
+        </property>
+        <property name="toolTip">
+         <string>+AppVMs +Standalones&lt;br&gt;-Disposables -ServiceVMs</string>
```

Comment:
Maybe "Show AppVMs and Standalone VMs, hide disposable and service VMs"?

Reply:
> Maybe "Show AppVMs and Standalone VMs, hide disposable and service VMs"?

Yes. This is much much better. I did not like the old message personally.